src/capabilities.rs

```diff
@@ -470,6 +470,7 @@ pub const CAPABILITY_SEMANTIC_TOKENS: &str = "lsp-semantic-tokens";
 pub const CAPABILITY_SIGNATURE_HELP: &str = "lsp-signature-help";
 pub const CAPABILITY_TYPE_DEFINITION: &str = "lsp-type-definition";
 pub const CAPABILITY_WORKSPACE_SYMBOL: &str = "lsp-workspace-symbol";
+pub const CAPABILITY_DOCUMENT_LINKS: &str = "lsp-document-links";
```

Comment:
this should match the name of the command, so `lsp-document-link` (no s)

Actually we should probably lose the s almost everywhere in this PR
(making it `CAPABILITY_DOCUMENT_LINK`, `mod document_link`)
this should be consistent with most others like `document_symbol`.